.gitlab/source_test/linux.yml

```diff
@@ -35,7 +35,7 @@
     - inv -e test $FLAVORS --include-sds --race --profile --rerun-fails=2 --python-runtimes "$PYTHON_RUNTIMES" --coverage --cpus $KUBERNETES_CPU_REQUEST $EXTRA_OPTS --save-result-json $TEST_OUTPUT_FILE --junit-tar "junit-${CI_JOB_NAME}.tgz" --build-stdlib $FAST_TESTS_FLAG --test-washer
     # Upload coverage files to Codecov. Never fail on coverage upload.
     - export CODECOV_TOKEN=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $CODECOV_TOKEN_SSM_NAME)
-    - inv -e codecov || true
+    - inv -e codecov $COVERAGE_CACHE_FLAG --debug || true
```

Comment:
```suggestion
    - inv -e codecov $COVERAGE_CACHE_FLAG || true
```

Reply:
Thanks for catching this ! It's on purpose, since the feature is still experimental, I'll review it in another PR in a few days !